apps/storybook/src/forms/Field.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { Field, Input } from "@optiaxiom/react";
+import { IconCalendar } from "@tabler/icons-react";
+
+const meta: Meta<typeof Field> = {
+  component: Field,
+  parameters: {
+    layout: "centered",
+  },
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Field>;
+
+export const InputWithIcon: Story = {
+  render: () => (
+    <Field label="Form label" required>
+      <Input leftSection={<IconCalendar size="16" />} placeholder="With Icon" />
+    </Field>
+  ),
```

Comment:
let's use the `args` convention instead of `render` - that way the controls in storybook will be integrated into the story

```suggestion
  args: {
    children: <Input leftSection={<IconCalendar size="16" />} placeholder="With Icon" />,
    label: 'Form label',
    required,
  },
```

https://github.com/optimizely-axiom/optiaxiom/assets/758244/92f235b4-2e32-48ab-8522-4af7175cc124